org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/DocumentLifeCycleHandlerTest.java

```diff
@@ -117,15 +117,28 @@ public void testUnimplementedMethods() throws Exception {
 		ICompilationUnit cu = pack1.createCompilationUnit("F.java", buf.toString(), false, null);
 		openDocument(cu, cu.getSource(), 1);
 
-		buf = new StringBuilder();
+		List<Command> commands = getCodeActions(cu);
+		assertTrue(commands.size() == 1);
```

Comment:
never ever use assertTrue when assertEquals can do the job, you lose important diagnostic informations, i.e what was the actual value received.